R/cluster.R

```diff
@@ -131,13 +131,13 @@ setMethod("cluster", signature = c(x = "SummarizedExperiment"),
     if (.is_non_empty_string(MARGIN)) {
         MARGIN <- tolower(MARGIN)
     }
-    if (length(MARGIN) != 1L 
-        || !(MARGIN %in% c(1, 2, "features", "samples", "columns", 
```

Comment:
There is lots of removed spaces. It is good that there is no empty spaces, but it looks like these all lines have changed, so it is better to not remove those spaces